collections/chunked.ts

```diff
@@ -6,6 +6,8 @@
  * Example:
  *
  * ```typescript
+ * import { chunked } from "https://deno.land/std/collections/chunked.ts";
```

Comment:
Using import like this will cause to download latest `main` branch - it means that if in a PR you change an API the doctests might fail because `main` contains older version. 

I think we should use `import { chunked } from "./chunked.ts";` (though it's not so ergonomic, because if you want to copy-paste the example, you'd have to adjust the import statement.

@caspervonb could you double check and suggest the best solution?

Reply:
Should definitively be using _relative_ module specifiers, The source itself doesn't live on deno.land after all. If I haven't already documented this in the manual as the recommended way to import "self" I definitively intended to do so!

We can rewrite it to absolute urls tagged to the correct version when we render in deno.land's source view component.

Reply:
Relative specifiers make sense especially from the maintainer's viewpoint. Updated.